### PR TITLE
Cases Pagelinks

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Via frontmatter the author of a case can now define a link where a specific service or product has to link to. 

This gets translated, so on the German website the author must use real page paths: 

kwaliteitpaspoort to link to qualitaetspass for example. 

Part of #1053

### CODE REVIEW

File: source/cases/de/evangelisches-krankenhaus-oldenburg.html.markdown [hunk 1]
```diff
@@ -2,10 +2,12 @@
 title: "Evangelisches Krankenhaus Oldenburg"
 date: 2017-09-10 10:00 CEST
 services:
- - CAPP Bilden
- - CAPP Entwickeln
- - Qualitätspass
- - Qualitätsmonitor
+ - title: CAPP Bilden
+   link: capp-lms
```
Comment (sn3p) [inline source/cases/de/evangelisches-krankenhaus-oldenburg.html.markdown]: I'd include the `/` in the url to make it more clear/complete.

File: source/stylesheets/_posts.scss [hunk 2]
```diff
@@ -39,16 +39,18 @@ article.post {
...
-    font-weight: 700;
-    text-decoration: none;
-    transition: color 0.1s linear;
-
-    &:active,
-    &:focus,
-    &:hover {
-      color: darken($blog-link-color, 15%);
+  body:not(.cases) & {
+    a:not(.button) {
```
Comment (sn3p) [inline source/stylesheets/_posts.scss]: Would this work as well?

```css
body:not(.cases) a:not(.button) {
```

Comment (RonaldDijkstra) [reply source/stylesheets/_posts.scss]: `body:not(.cases) & a:not(.button)`

**Comment by RonaldDijkstra** [comment]
screen shot 2018-08-01 at 10 34 39

**Comment by RonaldDijkstra** [comment]
Hound...

File: source/stylesheets/_posts.scss (hunk 2 above)
Comment (sn3p) [reply source/stylesheets/_posts.scss]: Why is the `&` needed?

Comment (RonaldDijkstra) [reply source/stylesheets/_posts.scss]: Without it all blog links get the orange action-color and I do not want that at this point. They should be the blue action-color.

**Comment by RonaldDijkstra** [comment]
Needs rebase

**Comment by RonaldDijkstra** [comment]
Rebased. Dropped some commits which where not in scope for this PR and conflicted with `master`. 

Deployed again to staging.defacto.nl

File: source/stylesheets/_posts.scss (hunk 2 above)
Comment (RonaldDijkstra) [reply source/stylesheets/_posts.scss]: Probably some weird nesting going on here where I have to look into later.

**Comment by fatboypunk** [comment]
looks good to me, although i find the yellow links a bit hard to read

**Comment by RonaldDijkstra** [comment]
Noted @fatboypunk, I'll look into it later on.

**Pull request merged**